addons/hr_attendance/static/src/public_kiosk/public_kiosk_app.js

```diff
@@ -41,6 +41,7 @@ class kioskAttendanceApp extends Component{
             displayDemoMessage: browser.localStorage.getItem("hr_attendance.ShowDemoMessage") !== "false",
         });
         this.lockScanner = false;
+        this.pwaService = useService("pwa");
```

Comment:
Not sure about this?
why are we using this service?

Reply:
We were using it for the link generation of the install button but we don't need it anymore, as we are creating it separately.